meroxa/cli
# feat(resources): Add update resources cmd

# Description of change

Fixes: https://meroxa.atlassian.net/browse/PLATFORM-13

Needs: https://github.com/meroxa/meroxa-go/pull/13

- [x] Tested this works
- [x] Ship https://github.com/meroxa/meroxa-go/pull/13
- [x] Update reference to `meroxa-go@latest`
- [ ] Ship this PR.

# Type of change


# How was this tested?


# Additional references

**Steps to test this work:**

1. Check out this branch.
2. Run `GOPRIVATE="github.com" go get -u github.com/meroxa/meroxa-go@raul_update_resource`
3. `go mod vendor`
4. `make build`
5. Make sure you have resources `meroxa list resources`
6. Try to update one.
